projects/commudle-admin/src/app/feature-modules/notifications/components/notifications-list/notifications-list.component.ts

```diff
@@ -15,13 +16,32 @@ export class NotificationsListComponent implements OnInit, OnDestroy {
   ENotificationStatus = ENotificationStatus;
 
   subscriptions = [];
+  moment = moment;
 
   constructor(
     private notificationService: NotificationService,
     private notificationStateService: NotificationStateService,
   ) {}
 
-  ngOnInit(): void {}
+  ngOnInit(): void {
+    moment.locale('en', {
+      relativeTime: {
+        past: '%s',
+        s: '1s',
+        ss: '%ss',
+        m: '1m',
+        mm: '%dm',
```

Comment:
Let's create a separate function for this and call it in `onInit`

Reply:
done